python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -49,6 +49,44 @@
 PREEMPTIVE_RENEWAL_THRESHOLD = 0.9
 SHOTGUN_SSO_RENEWAL_INTERVAL = 5000
 
+# Some IdP will use JavaScript code which makes use of ES6. Our Qt4
```

Comment:
nit: please spell out what `IdP`means.

Reply:
Can we be more specific here? Eg. include a date for the breakage and which exact providers?
This will help people understand the backstory for this feature and potentially accelerate future fixes.
I think the more info we can include, the better (e.g. even details of the package change on the okta side that was causing the breakage etc)

Reply:
Unfortunately I have no specific infos to give (yet).
Not sure when the breakage occured for Okta (in the preview environment) or when it will be released to Production.